src/main/zip/info.xml

```diff
@@ -184,5 +184,8 @@
     <release-note plugin-version="22">
         RFE 142129 - Added Deployment-Policy-variables field to the Import (Deployment Policy Object) step.
     </release-note>
+    <release-note plugin-version="23">
+        RFE 143590  - Need Download Files plugin step to support downloading from multiple datapower directories.
```

Comment:
Please update with a more formal release-note that better represent the actual change. 

Example... `RFE ... - The Download Files step now supports downloading files from multiple directories.`